manifests/params.pp

```diff
@@ -52,6 +52,7 @@
   $environment = 'production'
   $gpgcheck    = true
   $version     = 'present'
+  $timeout     = 60
```

Comment:
I'm not sure I like this generic name here. I'd name it ```puppetmaster_timeout```. In puppetmaster itself the parameter should still be just timeout.